tests/test_examples.py

```diff
@@ -123,10 +123,6 @@
     "vae/ss_vae_M2.py --num-epochs=1 --enum-discrete=sequential",
     "vae/vae.py --num-epochs=1",
     "vae/vae_comparison.py --num-epochs=1",
-    pytest.param(
-        "cvae/main.py --num-quadrant-inputs=1 --num-epochs=1",
-        marks=pytest.mark.skip(reason="https://github.com/pyro-ppl/pyro/issues/3273"),
-    ),
```

Comment:
Could you replace these four lines with just the string
```
"cvae/main.py --num-quadrant-inputs=1 --num-epochs=1",
```
The pytest.mark.skip is just to skip the test, and we now want it to run.

Reply:
Added the line for the test.